cfme/tests/openstack/infrastructure/test_roles.py

```diff
@@ -21,67 +20,67 @@
          'CephStorage']
 
 
+@pytest.yield_fixture(scope="module")
+def roles(appliance, provider):
+    collection = DeploymentRoleCollection(appliance=appliance)
+    roles = collection.all(provider)
+    yield roles if roles else pytest.skip("No Roles Available")
```

Comment:
**REQUIRED**: No, we shouldn't just skip if we can't find a role, the test should add one

Reply:
Apologize to say you @psav but In case of Deployment Role, Creation method not supported by our UI [No add option available]. Is there any other way to add Deployment role? please suggest me. 

Yes, I can avoid skip, as per my observation, we found default deployment roles always in cloud infra provider.

Reply:
TODO added @psav